terra-workspace-manager-performance/src/gatling/resources/application.conf

```diff
@@ -0,0 +1,15 @@
+dev {
+  terra {
+    env {
+      wsmBaseUrl = "WSM_BASE_URL"
+    }
+  },
+  sam {
+    basePath = "https://sam.dsde-dev.broadinstitute.org",
+    samBillingProject = "sam-perf-test-user",
+    email = "william.thunderlord@test.firecloud.org",
+    firecloudServiceAccount = "FIRECLOUD_SERVICE_ACCOUNT_CREDS",
+    firecloudServiceAccountPath = "FIRECLOUD_SERVICE_ACCOUNT_PATH",
+    serviceAccountFilePath = "SERVICE_ACCOUNT_FILE_PATH"
```

Comment:
Not critical - but I was expecting to see actual "accounts" and "paths" as values of these variables. Maybe these variable names can be updated?

Reply:
> Not critical - but I was expecting to see actual "accounts" and "paths" as values of these variables. Maybe these variable names can be updated?

Ahh this is pulling from an env variable. Nevermind.

Reply:
Cool